example/src/styles.css

```diff
@@ -1,3 +1,25 @@
 body {
+  margin-top: 60px;
```

Comment:
As shown in this file, additional CSS has become needed to fix menu's position and set appropriate margins for layout. But these styles could be injected automatically(or optionally) by cms-ui library. Should users do these styling manually?

Reply:
In my opinion, the case of custom menu styling is unlikely. There will be only cases of showing or hiding the menu. So, built-in styling is okay. If optional custom styling is offered, I think it's the best, but not necessary.